x/auth/types/account_retriever_test.go

```diff
@@ -11,6 +11,14 @@ import (
 )
 
 func TestAccountRetriever(t *testing.T) {
+	t.Skip("skipping because as not relevant to Heimdall (no delegation)")
```

Comment:
```suggestion
	t.Skip("skipping as not relevant to Heimdall (no delegation)")
```

Reply:
Ops, gonna go with the same comment we used for all the others here
`skipping test as not relevant to Heimdall (reason)`